apps/platform_runner/docker-compose.yml

```diff
@@ -27,6 +27,8 @@ services:
     logging:
       driver: none
     image: smartcitiesdata/postgres-testo:development
+    volumes:
+      - "./scripts/local_postgres_setup/:/docker-entrypoint-initdb.d"
```

Comment:
this will need to change if you go with the next suggestion

Reply:
Yep, already caught that. I thought my last commit included both path changes for local testing.